### DISCUSSION TITLE
How to enable css/scss/less language support in lazyvim.

### DISCUSSION THREAD

**Discussion by rick-25** [LazyVim/LazyVim#2492] [Q&A] [answered]
I am trying to add stylesheet support in lazyvim fresh install, please help me...
`plugins/cssls.lua`
```
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      servers = {
        cssls = {},
      },
    },
  },
}

**Comment by madflow** [comment]
Unclear what you are asking for. Maybe also add the Treesitter support?

```
return {
  {
    "nvim-treesitter/nvim-treesitter",
    opts = {
      ensure_installed = {
...
        "css",
...
```

https://www.lazyvim.org/plugins/treesitter

**Reply by rick-25** [reply]
I just installed the lazyvim config and I want those fancy autocompletion like vscode in css/scss files, I tried adding "css" in ensure_installed table it's not working.

**Reply by dpetka2001** [reply]
Can you show `:LspInfo` from the `.css` file?

**Reply by rick-25** [reply, marked as answer]
Got the lsp working by bumping the node version from 14 to 18.

**Reply by dpetka2001** [reply]
Then choose your own answer as the suggested answer, so that other people who might come across the same problem would know what to do.
